apps/server/src/modules/board/repo/recursive-delete.vistor.ts

```diff
@@ -100,6 +101,13 @@ export class RecursiveDeleteVisitor implements BoardCompositeVisitorAsync {
 		await this.visitChildrenAsync(externalToolElement);
 	}
 
+	async visitCollaborativeTextEditorElementAsync(
+		collaborativeTextEditorElement: CollaborativeTextEditorElement
+	): Promise<void> {
+		this.deleteNode(collaborativeTextEditorElement);
+		await this.visitChildrenAsync(collaborativeTextEditorElement);
```

Comment:
A CollaborativeTextEditorElement is not allowed to have children. Why call this method then?

Reply:
Low priority.